packages/desktop-client/src/hooks/useSelected.tsx

```diff
@@ -251,8 +251,8 @@ export function useSelectedDispatch() {
   return useContext(SelectedDispatch);
 }
 
-export function useSelectedItems() {
-  return useContext(SelectedItems);
+export function useSelectedItems<Item>() {
+  return useContext(SelectedItems) as Set<Item>;
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏manual type coercions lead to subtle bugs that are hard to spot.

Could we instead change line 248 to be

```
const SelectedItems = createContext<Set<string>>(null);
```

Reply:
I see your point, but I was concerned that the different usages of `SelectedItems` wouldn't all be `string`s. I'll double check and fix if they are